src/App.jsx

```diff
@@ -102,63 +104,84 @@ export default class App extends Component {
     })
   }
 
+
   ehrLaunch(isContainedQuestionnaire, questionnaire) {
-    // Temporary indication before full supports for relaunch is implemented
-    if(!this.appContext.request) {
-      alert("Supports for relaunch will be added in the near future!");
-      this.consoleLog("Supports for relaunch will be added in the near future!", "errorClass");
-      return;
+    let acOrder = this.appContext.order;
+    let acCoverage = this.appContext.coverage;
+    let acQuestionnaire = this.appContext.questionnaire;
+    let acResponse = this.appContext.response;
+    if(acOrder && acCoverage && !acQuestionnaire && !acResponse) {
+      // TODO: There's an additional case where you could launch
+      // with just the order/coverage by invoking the operation
+      // but I think the endpoint extension on coverage which
+      // would facilitate that is going away in ballot.
+      searchByOrder(acOrder, this.smart).then((res) => {
+        // TODO: Don't know how to deal with multiple QRs
+        // Let user pick with a UI?  Force orders to 
+        // uniquely identify QRs?  
+        // for now just pick the first one
```

Comment:
It probably makes sense for the user to pick the QRs as one order could end up with multiple QRs for different forms in DME order cases.